src/pull.rs

```diff
@@ -1,14 +1,32 @@
 use anyhow::Result;
+use indicatif::{ProgressBar, ProgressStyle};
 use policy_evaluator::policy_fetcher::{
     fetch_policy, policy::Policy, sources::Sources, PullDestination,
 };
+use std::time::Duration;
```

Comment:
nitpick: can you move the `std` include at the top of the file, then leave one empty line and have the other imports?

```suggestion
use std::time::Duration;

use anyhow::Result;
use indicatif::{ProgressBar, ProgressStyle};
use policy_evaluator::policy_fetcher::{
    fetch_policy, policy::Policy, sources::Sources, PullDestination,
};
```

This is best practice used by Rust projects to handle imports. I acknowledge this is something we are not doing everywhere, this is something we're slowly fixing.

Reply:
Ok, I will keep in mind next time.